examples/animation-controls/animation-controls.js

```diff
@@ -0,0 +1,63 @@
+
+animationNames = {
+  attack: 'Armature|TRex_Attack',
+  death: 'Armature|TRex_Death',
+  idle: 'Armature|TRex_Idle',
+  jump: 'Armature|TRex_Jump',
+  run: 'Armature|TRex_Run',
+  walk: 'Armature|TRex_Walk',
+};
+
+
+
+updateAnimationMixer = () => {
+
+  const data = {}
+  data.clip = "none"
+  Object.entries(animationNames).forEach((name) => {
+
+    el = document.getElementById(name[0])
+    if (el.checked) {
+      data.clip = name[1]
+    }
+  })
+
+  const getValue = (key) => {
+    const value = document.getElementById(key).value
+
+    if (AFRAME.components['animation-mixer'].schema[key].type === 'number' && isNaN(value)) {
+      return;
+    }
+    data[key] = value
+  }
+  getValue("duration")
+  getValue("clampWhenFinished")
+  getValue("crossFadeDuration")
+  getValue("loop")
+  getValue("repetitions")
+  getValue("timeScale")
+  getValue("startAt")
```

Comment:
I'm not super fan of this code. The name getValue is strange here for a function that doesn't return anything but set the value in a data object in the scope outside of the function.
Please consider something like that, I find it less confusing:
```js
const fieldKeys = ["duration", "clampWhenFinished", "crossFadeDuration", "loop", "repetitions", "timeScale", "startAt"]; // outside of updateAnimationMixer, no need to create the array every time you change the form.

fieldKeys.forEach((key) => {
    const value = document.getElementById(key).value

    if (AFRAME.components['animation-mixer'].schema[key].type === 'number' && isNaN(value)) {
      return;
    }
    data[key] = value
})
```